x-pack/plugins/threat_intelligence/common/types/indicator.ts

```diff
@@ -68,13 +67,12 @@ export const IndicatorFieldEventEnrichmentMap: { [id: string]: string[] } = {
   [RawIndicatorFieldId.FileSha3512]: ['file.hash.sha3-512'],
   [RawIndicatorFieldId.FileSha512224]: ['file.hash.sha512/224'],
   [RawIndicatorFieldId.FileSha512256]: ['file.hash.sha512/256'],
-  [RawIndicatorFieldId.FileSSDeep]: ['file.ssdeep'],
-  [RawIndicatorFieldId.FileTlsh]: ['file.tlsh'],
-  [RawIndicatorFieldId.FileImpfuzzy]: ['file.impfuzzy'],
-  [RawIndicatorFieldId.FileImphash]: ['file.imphash'],
-  [RawIndicatorFieldId.FilePehash]: ['file.pehash'],
-  [RawIndicatorFieldId.FileVhash]: ['file.vhash'],
-  [RawIndicatorFieldId.FileTelfhash]: ['file.elf.telfhash'],
+  [RawIndicatorFieldId.FileSSDeep]: ['file.hash.ssdeep'],
```

Comment:
same here, modifying following [this](https://docs.google.com/spreadsheets/d/1oYx9CQodKVmr0OtlJcjm1mNoUBlDcbHai0UYxtP8YzY/edit#gid=1814426429)